File: set_1/arrays/javascript/2055e568-b0b2-4a67-a761-f7714b383276.js
const arr = [2, 3, 2, 4, 4, 3, 7, 1, 9, 1, 7, 23];

// Solution 1:
const findSingleOccuringElem1 = (arr) => {
    const arrayLength = arr.length;
    let count;
    let singleOccuringElems = []
    for (let i = 0; i < arrayLength; i++) {
        count = 0;
        for (let j = 0; j < arrayLength; j++) {
            if (arr[i] === arr[j]) {
                count++;
            }
        }
        if (count === 1) {
            singleOccuringElems.push(arr[i]);
        }
    }
    return singleOccuringElems.length > 0 ? singleOccuringElems : "None";
}
const resp1 = findSingleOccuringElem1(arr);
console.log(`The element(s) which occurs only once --> ${resp1}`);

File: set_1/arrays/python/2055e568-b0b2-4a67-a761-f7714b383276.py
arr = [2, 3, 2, 4, 4, 3, 7, 1, 9, 1, 7, 0]


def findSingleOccuringElem(arr):
    arr_length = len(arr)
    single_occuring_elems = []
    for i in range(arr_length):
        count = 0
        for j in range(arr_length):
            if (arr[i] == arr[j]):
                count += 1
        if (count == 1):
            single_occuring_elems.append(arr[i])
    if len(single_occuring_elems) > 0:
        return single_occuring_elems
    return "None"


resp = findSingleOccuringElem(arr)
print(f"The element(s) which occurs only once --> {resp}")
